prow/auth-postsubmit.sh

```diff
@@ -63,3 +63,22 @@ fi
 
 echo "=== Running e2e Tests ==="
 bin/e2e.sh --tag ${GIT_SHA} --hub 'gcr.io/istio-testing'
+RET=$?
```

Comment:
You don't need to do that. At the beginning of the file we do 
set -e
Which will exit on failure.